start_kafka.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+BUILD_DIR=$(pwd)
```

Comment:
:+1:  I had done this locally and never actually committed it.  I have a couple local kafka clusters that I have to spin up/down depending on what I'm working on.